test/e2e/aks_machinepools.go

```diff
@@ -47,109 +47,92 @@ func AKSMachinePoolSpec(ctx context.Context, inputGetter func() AKSMachinePoolSp
 	input := inputGetter()
 	var wg sync.WaitGroup
 
-	originalReplicas := map[types.NamespacedName]int32{}
```

Comment:
Most of the changes in this file are indentation, so I'd suggest ticking the "hide whitespace" checkbox.